lua/lint.lua

```diff
@@ -304,7 +319,9 @@ function M.lint(linter, opts)
     stdio = { stdin, stdout, stderr },
     env = env,
     cwd = opts.cwd or linter.cwd or vim.fn.getcwd(),
-    detached = false
+    -- Linter may launch child processes so set this as a group leader and
+    -- manually track and kill processes as we need to.
+    detached = true
```

Comment:
I think this would cause issues on windows. See https://github.com/mfussenegger/nvim-lint/commit/4fb01e09af04e2ec953aa177f38ea6ded62e97eb


Would changing from sigterm to sigint be good enough?

Reply:
Hard to say, sigint is a much weaker signal so I would say we should ensure anything is closed with either sigterm or sigkill.

If windows has issues then we can force `detached` to false on that platform. It most definitely doesn't follow the same semantics anyway.

Reply:
I've updated so we do not detach on windows.